### PR TITLE
Add the guide for The Lounge

### LINKED ISSUE #206: Add guide for The Lounge
The lounge is a community driven always on web client for IRC.

The website can be found at https://thelounge.chat/.

I am currently working on this guide.

### PR DESCRIPTION
**Pull request by Roang-zero1**
This guide describes how to set up The Lounge as an IRC web client on Uberspace 7.

This closes #206.

### CODE REVIEW

**Comment by Roang-zero1** [comment]
Hi, I've updated the guide to fix some small issues that slipped though on my last review.

File: source/guide_the-lounge.rst [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+For a full list of all configuration options see `the official documentation`_.
+
+Creating users
+--------------
+
+Since we do not run a public server we have to add users before we can use The Lounge:
+
+::
+
+ thelounge add isabell
```
Comment (SalocinHB) [inline source/guide_the-lounge.rst]: The `~/.thelounge/users/` directory needs to be created before adding users

```
[nico@stardust ~]$ thelounge add nico
2018-10-09 17:24:33 [ERROR] /home/nico/.lounge/users does not exist.
```

Comment (Roang-zero1) [reply source/guide_the-lounge.rst]: The first start of the service will create all necessary directories, so I moved the user management section to the **Finishing installation** section.

**Comment by Roang-zero1** [comment]
I've expanded the user section and added emphasis to lines that require user action

**Pull request merged**